.github/workflows/release.yml

```diff
@@ -0,0 +1,148 @@
+name: Create release
+on: [workflow_dispatch]
+
+jobs:
+  release:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.set_variables.outputs.version }}
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - uses: actions/setup-python@v5
+      with:
+        python-version: "3.8"
+
+    - name: Install Hatch
+      run: pipx install hatch
+
+    - name: Set variables
+      id: set_variables
+      run: |
+        tag=$(git for-each-ref refs/tags --sort=-creatordate --format='%(refname:strip=2)' --count=1)
+        cp README.md plugin/
+        cd plugin/
+        echo "::group::Variables"
+        cat << EOF | tee -a "$GITHUB_OUTPUT"
+        tag=${tag}
+        version=v$(hatch project metadata | jq -r .version)
+        EOF
+        echo "::endgroup::"
+    - name: Bundle and create release
+      env:
+        GH_TOKEN: ${{ github.token }}
+        tag: ${{ steps.set_variables.outputs.tag }}
+        version: ${{ steps.set_variables.outputs.version }}
+      if: env.tag != env.version
+      run: |
+        cd plugin/
+        project_name="$(hatch project metadata | jq -r .name)"
+        mkdir bundle/
+        cp -r yt_dlp_plugins bundle/
+        cd bundle/
+        find . -type f -name '*.py[co]' -delete -o -type d -name __pycache__ -delete
+        zip -9 --recurse-paths "${project_name}" *
+        printf "## What's Changed\n\n" >> CHANGELOG.md
+        git log '--pretty=* %h(by %an\<%ae\>)%n```md%n%B%n```' --reverse ${tag}...HEAD >> CHANGELOG.md
```

Comment:
The syntax highlighting breaks here. It looks fine to me, but you've made sure that this part works right?

Reply:
Use github.dev to see syntax highlighting. It is working, see
> Examples (released by [f19300f](https://github.com/Brainicism/bgutil-ytdlp-pot-provider/commit/f19300fe510a19156a7e270cfe46524d8d2baa83)):
> 
> * Github release (plugin): [definitely-not-bgutil-ytdlp-pot-provider-to-distinguish-from-it v0.2.16](https://github.com/grqz/bgutil-ytdlp-pot-provider/releases/tag/v0.2.16)

The changelog currently looks like this. Do we need to change it to the one like in your repo? It's cumbersome to get github user login name in this script

Reply:
I think it's fine, not a huge deal with the usernames aren't there for now

Reply:
```sh
gh pr list --json author,number,title -q '.[]| "* \(.title) " + "by \(.author.login)" + " (#\(.number|tostring))"' -R Brainicism/bgutil-ytdlp-pot-provider --state merged
```
this should work with `--since` and the last tag's creatordate

Reply:
Now it looks like the one in your repo
v0.2.24: https://github.com/grqz/bgutil-ytdlp-pot-provider/releases/tag/v0.2.24